obfuscator-core/src/main/java/me/superblaubeere27/jobf/JObf.java

```diff
@@ -124,7 +126,7 @@ public static void main(String[] args) throws Exception {
 
             try {
                 if (Utils.isWindows()) {
-                    UIManager.setLookAndFeel("com.bulenkov.darcula.DarculaLaf");
+                    FlatDarculaLaf.setup();
```

Comment:
Only applied to windows, as it seems it is project preference to use native look and feel when using other Operating Systems